corehq/apps/accounting/models.py

```diff
@@ -2131,6 +2131,13 @@ class Invoice(InvoiceBase):
 
     class Meta(object):
         app_label = 'accounting'
+        constraints = [
+            models.UniqueConstraint(
+                fields=['subscription', 'date_start', 'date_end'],
+                name='unique_invoice_per_subscription_period',
+                condition=models.Q(is_hidden_to_ops=False)
```

Comment:
Should there be a `duplicate_invoice_id IS NULL` condition too?
```suggestion
                condition=models.Q(is_hidden_to_ops=False, duplicate_invoice_id__isnull=True)
```

Reply:
Hi Daniel, the latest commit is just a merge from master. I haven't pushed my changes yet. :joy: I'll ping you when it's ready for review. I'll change the PR to draft now.